models/pathology_nuclei_segmentation_classification/docs/README.md

```diff
@@ -0,0 +1,124 @@
+# Model Overview
+A pre-trained model for simultaneous segmentation and classification of nuclei within multitissue histology images based on CoNSeP data. The details of the model can be found in [1]
+
+## Workflow
+
+The model is trained to simultaneous segment and classify nuclei. Training is done via a two-stage approach. First initialised the model with pre-trained weights on the [ImageNet dataset](https://ieeexplore.ieee.org/document/5206848), trained only the decoders for the first 50 epochs, and then fine-tuned all layers for another 50 epochs.
+
+- Each user is responsible for checking the content of models/datasets and the applicable licenses and determining if suitable for the intended use.The license for the pre-trained model used in examples is different than MONAI license. Please check the source where these weights are obtained from: https://github.com/vqdang/hover_net#data-format
+
+`pretrained_model` is "https://drive.google.com/u/1/uc?id=1KntZge40tAHgyXmHYVqZZ5d2p_4Qr2l5&export=download" which will be used in bash code below.
+
+![Model workflow](https://ars.els-cdn.com/content/image/1-s2.0-S1361841519301045-fx1_lrg.jpg)
+
+## Data
+
+The training data is from https://warwick.ac.uk/fac/cross_fac/tia/data/hovernet/.
```

Comment:
we need a `data_license.txt` in `docs`. Please take: https://github.com/Project-MONAI/model-zoo/blob/dev/models/pathology_tumor_detection/docs/data_license.txt for reference.

Reply:
The license of CoNSeP dataset is Apache2.0, the same as the bundle, do we still need to add the `data_license.txt` file?

Reply:
I think it is needed since apache 2.0 is also a license. You can do sth like: https://github.com/Project-MONAI/model-zoo/blob/dev/models/spleen_ct_segmentation/docs/data_license.txt